Examples/SessionTools-macOSExample/LoginViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  LoginViewController.swift
+//  SessionTools-macOSExample
+//
+//  Created by Ryan Gant on 1/15/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
```

Comment:
Let's change the copyright to use Bottle Rocket Studios.
```
//  Copyright (c) 2019 Bottle Rocket Studios. All rights reserved.
```

Reply:
Done.